homeassistant/components/venstar/climate.py

```diff
@@ -275,11 +275,11 @@ def set_temperature(self, **kwargs):
             set_temp = self._set_operation_mode(operation_mode)
 
         if set_temp:
-            if operation_mode == self._client.MODE_HEAT:
+            if operation_mode in (self._client.MODE_HEAT, HVAC_MODE_HEAT):
```

Comment:
Operation mode should only be a home assistant specific constant, not a constant on the client.

Please change that.